src/controllers/controlpickermenu.cpp

```diff
@@ -645,6 +645,37 @@ ControlPickerMenu::ControlPickerMenu(QWidget* pParent)
                tr("Replace Auto DJ Queue with selected tracks"),
                libraryMenu, false, m_libraryStr);
     libraryMenu->addSeparator();
+    addControl("[Library]",
+            "search_history_next",
+            tr("Select next search history"),
+            tr("Selects the next search history entry"),
+            libraryMenu,
+            false,
+            m_libraryStr);
+    addControl("[Library]",
+            "search_history_prev",
+            tr("Select previous search history"),
+            tr("Selects the previous search history entry"),
+            libraryMenu,
+            false,
+            m_libraryStr);
+    addControl("[Library]",
+            "search_history_move",
```

Comment:
"move" is kinda odd here... how about
```suggestion
            "search_history_select",
```